react-components/src/components/BundleProductOptions/bundleProductOptions.js

```diff
@@ -277,6 +297,19 @@ const BundleProductOptions = () => {
                         <span>{intl.formatMessage({ id: 'product:add-item', defaultMessage: 'Add to Cart' })}</span>
                     </span>
                 </button>
+                <button
+                    className="button__root_normalPriority button__root clickable__root"
+                    type="button"
+                    onClick={addToWishlist}>
+                    <span className="button__content">
+                        <span>
+                            {intl.formatMessage({
+                                id: 'product:add-item-wishlist',
+                                defaultMessage: 'Add to Wish List'
```

Comment:
In the productrecs you used `Add to Wishlist` and `productrecs:add-to-wishlist`. Better use `product:add-to-wishlist` here as well. Regarding the label I would keep `Add to Wish List`.